docs/assets/templates/tools/template-small.dockstore.yml

```diff
@@ -2,7 +2,7 @@ version: 1.2
 tools:
   - subclass: CWL
     primaryDescriptorPath: <String>
-    testParameterFiles: <String Array>
+    testParameterFiles: <String Array>  # Must be a YAML array of strings
     authors:
       - name: <String>
```

Comment:
Here's a precedent for having an actual array, although this one's not a simple list of strings.